src/app/components/Card.tsx

```diff
@@ -0,0 +1,61 @@
+interface Props {
+  cardType: number;
+}
+
+const cardClassnames: Record<string, string> = {
+  container: 'p-4 mb-5 bg-teal text-white rounded-lg',
+  title: 'text-xl font-black',
+  data: 'font-normal',
+  date: 'font-extralight',
+};
+
+export default function Card({ cardType }: Props): JSX.Element {
+  let el: JSX.Element = <></>;
+  switch (cardType) {
+    case 0:
+      el = juzCard();
+      break;
+    case 1:
+      el = surahCard();
+      break;
+    case 2:
+      el = ayahCard();
+      break;
+  }
+
+  return el;
+}
+
+function juzCard(): JSX.Element {
+  return (
+    <div className={cardClassnames.container}>
+      <p className={cardClassnames.title}>Juz 28</p>
+      <p className={cardClassnames.data}>Murojaah by using Memory</p>
+      <p className={cardClassnames.data}>Total Murojaah is 100</p>
+      <p className={cardClassnames.date}>Sat, Feb 10 &apos;24</p>
+    </div>
+  );
+}
```

Comment:
this is what i had in mind for reusabled `Card`
```
const Card = ({title, contents, date}: Props) => (
    <div className={cardClassnames.container}>
      <p className={cardClassnames.title}>{title}</p>
      {contents.map(content => (
        <p className={cardClassnames.data}>{content}</p>
      )}
      <p className={cardClassnames.date}>{date}</p>
    </div>
)
```
and then in page.tsx we can create these:
```
const JuzCard = () => (
  <Card
    title="Juz 28"
    contents={["Murojaah by using Memory", "Total Murojaah is 100"]}
    date="Sat, Feb 10 &apos;24"
    />
)

...
const SurahCard ...
const AyahCard ...
```

Reply:
already updated on https://github.com/RR-Ardhika/murojaah/pull/3/commits/69019b3f97b3a880a20250ac7d32f9eb7385a0dc